omniauth-saml.gemspec

```diff
@@ -12,11 +12,12 @@ Gem::Specification.new do |gem|
   gem.homepage      = 'https://github.com/omniauth/omniauth-saml'
 
   gem.add_runtime_dependency 'omniauth', '~> 1.3'
-  gem.add_runtime_dependency 'ruby-saml', '~> 1.1', '>= 1.1.1'
+  gem.add_runtime_dependency 'ruby-saml', '~> 1.1'
```

Comment:
Can you revert this line? There were some issues with ruby-saml 1.1.0, so I consider this a regression.